client/src/services/club.js

```diff
@@ -26,6 +26,15 @@ async function getClubs() {
   return response.json();
 }
 
+async function getFavoriteClubs() {
+  const response = await fetch(`${apiUrl}/clubs`, {
```

Comment:
I think this endpoint should be filtered

Reply:
Yep, you right, I will fix right now.